### ISSUE TITLE
Add a list of tools I use everyday

### ISSUE THREAD

**Issue by diego3g** [diego3g/faladev#20]
Add a doc with the tool list a use everyday like Cron, CleanShot, Raycast, etc...

**Comment by mateusabelli** [comment]
Hello, sure I'd like to work on it!

I just have a few questions, is it supposed to be included inside your Explorer component on the page? 

![image](https://user-images.githubusercontent.com/43862225/221374998-140e7d3c-5a25-4f41-a6af-43f5e2b71ba1.png)

If that's it, could you please provide or point me to where I can find your tool list to be included?

**Comment by diego3g** [comment]
Yeah, probably! We are currently moving to use Notion as the back-end for this kind of page on #16 so I think we should wait the merge of this PR so it will be easier to add more pages. What do you think?

**Comment by mateusabelli** [comment]
Hello there! sure, no problem.

Just one question, after it gets merged only _authed_ personal would be able to create/modify the written content, in this case this issue would become obsolete?

**Comment by diego3g** [comment]
@mateusabelli we will still need to create the page inside the application but the content will be inside Notion. The content of the page itself will be managed by me, but if you want to create it with your ideas you can already do it and share with me.
